### DISCUSSION TITLE
Find the parent type name (not field)

### DISCUSSION THREAD

**Discussion by LunaticMuch** [graphql-hive/graphql-eslint#1135] [Q&A]
Hi,

I am trying to configure a check in naming convention to verify that all my mutations returns a type which ends with `Payload`.

However, using `FieldDefinition[parent.name.value=Mutation]` does not work, because it would applied to the field name. I tried also with `ObjectTypeDefinition[parent.name.value=Mutation]` but does not work as well.

Which option should actually go back to the parent name?

**Comment by dimaMachina** [comment]
Hi, I think `ObjectTypeDefinition[name.value=Mutation]` and `ObjectTypeExtension[name.value=Mutation]` should work

**Reply by dimaMachina** [reply]
I was wrong, for the following check

<img width="710" alt="image" src="https://user-images.githubusercontent.com/7361780/187161182-50f83f38-8677-4f26-b10b-b35258020966.png">

you selector must be `FieldDefinition[parent.name.value=Mutation] > .gqlType Name`
